packages/web/src/pages/app/repositories.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { NextPage } from 'next';
+import {
+  Input,
+  Button,
+  Link,
+  Heading,
+  useTheme,
+  VStack,
+  ListItem,
+  Box,
+  SimpleGrid,
+  FormControl,
+} from '@chakra-ui/react';
+import { AppLayout } from '../../components/Layout';
+import { RepoBox } from '../../components/Repos';
+
+export interface RepoList {
+  id: React.Key | undefined;
+  name: string;
+  html_url: string;
+  stargazers_count: Number;
+  language: string;
+  description: string | null;
+}
+
+const SearchBar: NextPage = () => {
```

Comment:
Make sure the name of the component specifies what it is, at first I thought this was just the search bar component

```suggestion
const RepositoriesPage: NextPage = () => {
```

Reply:
name changed Projects